linux_sql/README.md

```diff
@@ -9,25 +9,26 @@ The Monitoring agent sends the data to a database for storage and further analys
 Technologies used : Docker,Git,Linux,PostgreSQL,Bash
 
 # Quick Start
```

Comment:
![image](https://user-images.githubusercontent.com/58951265/132912841-c616ea1f-3d71-491c-a0b1-be18e0fab84b.png)
double check rendered version 

https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara/blob/74dc5fc51a4ef39794b1d3a04b650395a831a752/linux_sql/README.md


bullet points 
`- item1 (there is a space after the dash)`
`- item2`